app/enterprise/1.5.x/kong-for-kubernetes/deployment-options.md

```diff
@@ -0,0 +1,155 @@
+
+* Instead of using Kong Manager, proxy configuration is normally managed by the
+  controller, and you would provide configuration via Kubernetes resources.
+* Because the controller creates proxy configuration on behalf of users, you do
+  not need to interact with the Admin API directly. Kong's own RBAC
+  implementation isn't required for typical Kong for Kubernetes deployments, as
+  they do not expose the Admin API; only the controller can access it.
+  Kubernetes-level RBAC rules and namespaces should be used to restrict what
+  configuration administrators can create.
+* Ingress controller instances create configuration in a single workspace only
+  (`default` by default). To use multiple workspaces, deploy
+  multiple controller instances, setting the `CONTROLLER_KONG_WORKSPACE`
+  environment variable to the workspace that instance should use. These
+  instances should set `CONTROLLER_INGRESS_CLASS` to unique values for each
+  instance to avoid creating duplicate configuration in workspaces. Note that
+  if controller instances are deployed outside the Kong pod the Admin API must
+  be exposed, and users should enable RBAC with workspace admin users for the
+  controllers.  Set `CONTROLLER_KONG_ADMIN_TOKEN` to the RBAC user's token.
+* The controller cannot manage configuration for the features above: it cannot
+  create workspaces, Dev Portal content, admins, etc. These features must be
+  configured manually through the Admin API.
+
+### Plugin compatibility
+
+Not all plugins are compatible with DB-less operation, therefore not all
+plugins are available in the `kong-enterprise-k8s` image. Review the [list of
+supported plugins][supported-plugins] to see if you require a plugin that needs
+a database.
+
+Third-party plugins are generally compatible with DB-less as long as they do
+not create custom entities (i.e. they do not add new entities that users can
+create and modify through the Admin API).
+
+### Manual configuration
+
+DB-less configuration must be supplied as a complete unit: it is not possible
+to add or modify entities individually through the Admin API, or provide
+partial configuration that is added to existing configuration. As such, all
+configuration must be sourced from Kubernetes resources so that the ingress
+controller can render it into a complete configuration.
+
+On database-backed deployments, users can create or modify configuration
+through the Admin API. The ingress controller uses a tag (set by the
+`CONTROLLER_KONG_ADMIN_FILTER_TAG` environment variable) to to identify
+configuration that it manages. While the controller will revert changes to
+configuration with its tag, other configuration is left as-is.
+
+Although database-backed deployments can use controller-generated and
+manually-added configuration simultaneously, Kong's recommended best practice
+is to manage as much configuration through Kubernetes resources as possible.
```

Comment:
With the exception of consumers. 

Can we explain how some use-cases require consumers to be dynamically created and consumers can exist with a large cardinality, and in that case one should use Admin API instead of k8s CRDs?